projects/ng-core-tester/src/app/home/home.component.ts

```diff
@@ -14,17 +14,19 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { ApiService, DialogService, RecordSearchService, TranslateLanguageService } from '@rero/ng-core';
 import { ToastrService } from 'ngx-toastr';
 import { DocumentComponent } from '../record/document/document.component';
 import { NgxSpinnerService } from 'ngx-spinner';
+import { MenuService } from '../service/menu.service';
+import { MenuItem } from 'projects/rero/ng-core/src/lib/menu/menu-item';
```

Comment:
Should be imported from `@rero/ng-core`.

Reply:
done